sdk/tests/conformance2/textures/misc/tex-image-with-bad-args.html

```diff
@@ -65,6 +65,8 @@
     { internalformat: gl.RGBA, format: gl.RGBA, type: gl.HALF_FLOAT, errors: [gl.INVALID_OPERATION] },
     { internalformat: gl.LUMINANCE, format: gl.LUMINANCE, type: gl.FLOAT, errors: [gl.INVALID_OPERATION] },
     { internalformat: gl.LUMINANCE_ALPHA, format: gl.LUMINANCE_ALPHA, type: gl.HALF_FLOAT, errors: [gl.INVALID_OPERATION] },
+    { internalformat: 0x822A /*GL_R16_EXT*/ , format: gl.RED, type: gl.UNSIGNED_SHORT, errors: [gl.INVALID_VALUE] },
+    { internalformat: gl.RED, format: gl.RED, type: gl.UNSIGNED_SHORT, errors: [gl.INVALID_VALUE] },
```

Comment:
Thanks, but I also see this, from section 3.8.3 "Texture Image Specification" from page 127 of the ES 3.0.5 specification:

"Specifying a combination of values for format, type, and internalformat that is not listed as a valid combination in tables 3.2 or 3.3 generates the error INVALID_OPERATION."

The second call here with internalformat=GL_RED, format=GL_RED, and type GL_UNSIGNED_SHORT, can be considered to violate multiple rules -- e.g. if GL_RED should be considered GL_R8, the type would still be invalid (UNSIGNED_SHORT instead of UNSIGNED_BYTE).

Please change the second of these to also accept INVALID_OPERATION and then I'll merge it. Thanks.